bin/sparkling-env.sh

```diff
@@ -47,9 +53,13 @@ FAT_JAR="sparkling-water-assembly_$SCALA_VERSION-$VERSION-all.jar"
 FAT_JAR_FILE="$TOPDIR/assembly/build/libs/$FAT_JAR"
 major_version=`echo $VERSION | cut -d . -f 1,2`
 version_without_snapshot=`echo $VERSION | cut -d - -f 1`
-PY_EGG="h2o_pysparkling_${major_version}-${version_without_snapshot}-py2.7.egg"
+
+python_version=`python -c 'import sys; print(".".join(str(x) for x in sys.version_info[0:2]))'`
```

Comment:
What if user does not have python and he/she is not interested in pysparkling. We should handle this situation.